reana_commons/api_client.py

```diff
@@ -144,15 +152,13 @@ def submit(
         if htcondor_accounting_group:
             job_spec["htcondor_accounting_group"] = htcondor_accounting_group
 
-        response, http_response = self._client.jobs.create_job(job=job_spec).result()
-        if http_response.status_code == 400:
-            raise HTTPBadRequest(
-                "Bad request to create a job. Error: {}".format(http_response.data)
-            )
-        elif http_response.status_code == 500:
-            raise HTTPInternalServerError(
-                "Internal Server Error. Error: {}".format(http_response.data)
-            )
+        try:
+            response, http_response = self._client.jobs.create_job(
+                job=job_spec
+            ).result()
+        except Exception as e:
```

Comment:
Minor: I think it would be useful to catch specific Bravado exception for [HTTP requests errors](https://github.com/Yelp/bravado/blob/9e7008208de418c826e877b181b489dc5dec1d86/bravado/exception.py#L55). That way, in case the thrown exception doesn't match, we could log its message/stack trace too as a `logging.error` because that would mean something unexpected happened.